libs/shared/src/consts/providers/provider.interface.ts

```diff
@@ -16,6 +16,27 @@ export interface IConfigCredentials {
   type: string;
 }
 
+const mailConfigBase: IConfigCredentials[] = [
+  {
+    key: 'from',
+    displayName: 'From sender',
+    type: 'string',
+  },
+  {
+    key: 'senderName',
+    displayName: 'Sender name',
+    type: 'string',
+  },
+];
+
+const smsConfigBase: IConfigCredentials[] = [
+  {
```

Comment:
@djabarovgeorge did you something about AWS SNS eventually? I think it will apply this aswell no?

Reply:
What do you mean by 'apply this aswell', the SNS sms provider will be missing it because it does not have  concatenation of smsConfigBase.